src/entities/sql/connections.entity.ts

```diff
@@ -0,0 +1,27 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, Unique, CreateDateColumn, UpdateDateColumn, ManyToOne } from 'typeorm';
+import { UserEntity } from '.';
```

Comment:
what does the '.' mean?

Reply:
in entities/sql/index.ts we export our entities from there so we can import it to another file using "." instead of "./connections.entities.ts"